changes/ee/fix-15219.en.md

```diff
@@ -0,0 +1 @@
+Reduced the amount of logs generated by Kinesis Connector when a health check timeout occurs.  Also, when a health check timeout occurs for this Connector, we now mark it as `connecting` instead of `disconnected`, meaning that a full reconnect attempt will no longer be triggered by such timeouts.
```

Comment:
Kinesis ?

Reply:
d'oh, mixed the subjects :melting_face: :speak_no_evil: